### DISCUSSION TITLE
[20230609] Library shows single files in album view

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1350] [Bug Reports]
Hi,

my database contains three folders: Album, Sampler (mostly VA) and Singles. The latter do not contain an album tag but are still displayed in the album overview (without cover, now ~3k 'albums' vs 825 real ones). In 0603 it was fine.

Regards.
Oliver

**Comment by ceejayemm** [comment]
This seems to be the same as #1347.  You might want to continue there rather than start a new, duplicate thread ?

Chris
